### DISCUSSION TITLE
Grafana alerting and message template: Add "ago" Sprig function

### DISCUSSION THREAD

**Discussion by wengelbrecht-grafana** [grafana/grafana#52840] [Alerting]
Hello!

We're using Grafana Alerting and the messaging templates. 
We noticed that although the alerting messaging templates implements the Go language template system, it does not include the "ago" function, like this one: http://masterminds.github.io/sprig/date.html

It would be really nice to fire off an alert, with a message saying something like: 4h:37m ago
Right now we can only use the start and end functions, and subtract from the two but the output is pretty nasty. If we can use the "ago" function that formats the text really nicely.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
